docs/index.asciidoc

```diff
@@ -341,6 +341,18 @@ Events are uploaded in real-time without being stored to disk.
   for your use-case and consider using
   https://www.elastic.co/guide/en/logstash/current/persistent-queues.html[Logstash Persistent Queues]
 
+[id="plugins-{type}s-{plugin}-uploader_interval_secs"]
+===== `uploader_interval_secs`
+
+  * Value type is <<number,number>>
+  * Default value is `60`
+
+Uploader interval when uploading new files to BigQuery. Adjust time based
+on your time pattern (for example, for hourly files, this interval can be
+around one hour).
+
+**Deprecated:** this field is no longer used.
```

Comment:
You should use the deprecated notification here to insert our boilerplate statement about deprecations. See https://github.com/elastic/docs#section-notifications

Reply:
@dedemorton Is this a new requirement? The only places I have seen that style of deprecation warning in plugin docs, is when the [plugin itself](https://github.com/logstash-plugins/logstash-filter-anonymize/blob/6af07aaf6ad3e8b868212caca408b4b63b80a909/docs/index.asciidoc) was deprecated. For configuration options that have been deprecated, [this](https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/7c51ef41bf8e8295812f7029ac75d02b323b29d2/docs/index.asciidoc#document_type) seems to be the standard.

Reply:
@robbavey It's not a new requirement. We haven't used it consistently, but we probably should. The benefit is that we get deprecation statements that are consistent across the library, instead of each contributor coming up with their own language and convention for flagging deprecated options. Using the tag also give us the opportunity to change the way we flag deprecations (for example, if we decide we want deprecations to stand out more, we could change the look/feel in one place and have it show up everywhere). Plus having a flag (assuming me use it consistently) makes it easier to find (and remove) deprecated options.

Note that there's also a section in the docs readme that talks about inline deprecations (good for flagging options in tables).

Reply:
Thanks for the clarification @dedemorton